src/modules/publishing/PublishingDashboard.tsx

```
@@ -1,214 +1,415 @@
+
+            const matchesFilter = filterStatus === 'all' || release.status === filterStatus;
+
+            return matchesSearch && matchesFilter;
+        });
+    }, [releases, searchQuery, filterStatus]);
+
+    // Bulk Actions Handlers
+    const toggleSelection = useCallback((id: string) => {
+        setSelectedIds(prev => {
+            const newSet = new Set(prev);
+            const stats = React.useMemo(() => [
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)